src/code/z_actor.c

```diff
@@ -1549,22 +1549,35 @@ s32 func_8002F0C8(Actor* actor, Player* player, s32 flag) {
     return false;
 }
 
-u32 Actor_ProcessTalkRequest(Actor* actor, PlayState* play) {
-    if (actor->flags & ACTOR_FLAG_8) {
-        actor->flags &= ~ACTOR_FLAG_8;
+/**
+ * Provides a one-time notification to the actor that player has started talking to this specific actor.
+ */
+s32 Actor_TalkOfferAccepted(Actor* actor, PlayState* play) {
```

Comment:
how about this 
```c
/**
 * When a given talk offer is accepted, Player will set `ACTOR_FLAG_TALK` for that actor.
 * This function serves to acknowledge that the offer was accepted by Player, and notifies the actor
 * that it should proceed with its own internal procecees for handling dialogue.
 * 
 * @return  true if the talk offer was accepted, false otherwise
 */
 ```